src/state/get-droppable-over.js

```diff
@@ -1,22 +1,65 @@
 // @flow
-import { isPointWithin } from './is-within-visible-bounds-of-droppable';
+import { isPointWithinDroppable } from './is-within-visible-bounds-of-droppable';
+import { patch } from './position';
+import getDraggablesInsideDroppable from './get-draggables-inside-droppable';
 import type {
   DroppableId,
   Position,
+  DraggableDimension,
+  DraggableDimensionMap,
   DroppableDimensionMap,
   DroppableDimension,
 } from '../types';
 
-export default (
+const noPadding: Position = { x: 0, y: 0 };
```

Comment:
rather than using the name padding which is already a thing, let's use another name. What about 'buffer'? or 'extra'?

`noBuffer`